tests/TestCase/Model/Behavior/SearchBehaviorTest.php

```diff
@@ -36,6 +36,21 @@ public function searchConfiguration()
     }
 }
 
+class GroupsTable extends Table
+{
+
+    public function searchConfiguration()
+    {
+        $manager = new Manager($this);
+
+        return $manager
+            ->collection('frontend')
+            ->value('Groups.title')
```

Comment:
Remove the `Group.` alias. It's now done automatically in 2.x version of the plugin. The tests are failing because of that on PHP 5.4. They pass on 5.5+ in CakePHP 3.3 `aliasField()` was updated to ensure already aliased fields were not aliased again.